customdomain-terraform/modules/customdomain/main.tf

```diff
@@ -32,6 +32,20 @@ resource "aws_acm_certificate" "cert" {
   }
 }
 
+resource "aws_route53_record" "cert_validation" {
+  name    = aws_acm_certificate.cert.domain_validation_options.0.resource_record_name
+  type    = aws_acm_certificate.cert.domain_validation_options.0.resource_record_type
+  zone_id = var.zone_id
+  records = [aws_acm_certificate.cert.domain_validation_options.0.resource_record_value]
+  ttl     = 60
+}
+
+resource "aws_acm_certificate_validation" "cert-validation" {
+  provider                = aws.us-east-1
```

Comment:
Can we use `aws.eu-west-2`?

Reply:
Not sure, the ACM certificate needs to be in us-east-1 to work in CloudFront. I assume the validation resource will need to be too?

Reply:
According to AWS documentation, it says,

"AWS Region that You Request a Certificate In (for AWS Certificate Manager)

If you want to require HTTPS between viewers and CloudFront, you must change the AWS Region to US East (N. Virginia) in the AWS Certificate Manager console before you request or import a certificate.

If you want to require HTTPS between CloudFront and your origin, and you're using an ELB load balancer as your origin, you can request or import a certificate in any Region."

Source: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-requirements.html

Reply:
Okay, so that confirms that we do need the cert itself to be in us-east-1, what about the validation resource?